framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -863,29 +869,11 @@ public AnnotationMirror getAnnotationMirror(Tree tree, Class<? extends Annotatio
     }
 
     /**
-     * Determines the annotated type of an AST node.
-     *
-     * <p>
-     *
-     * The type is determined as follows:
-     * <ul>
-     *  <li>if {@code tree} is a class declaration, determine its type via
-     *    {@link #fromClass}</li>
-     *  <li>if {@code tree} is a method or variable declaration, determine its
-     *    type via {@link #fromMember(Tree)}</li>
-     *  <li>if {@code tree} is an {@link ExpressionTree}, determine its type
-     *    via {@link #fromExpression(ExpressionTree)}</li>
-     *  <li>otherwise, throw an {@link UnsupportedOperationException}</li>
-     * </ul>
+     * Returns an AnnotatedTypeMirror representing the qualified type of {@code tree}.
```

Comment:
It would be nice if this method also mentioned what defaulting and stub-file reading happens, like for fromElement below.